### ISSUE TITLE
Venezuela's idea modifiers are being added to the existing ones instead of being replaced.

### ISSUE THREAD

**Issue by hurafloyd** [deliciousmods/1956_beta#1872]
While going through Perez Jimenez' tree, when the "Gomez' Legacy" is developed the modifiers that are already set to "Widespread Corruption" are not being replaced by the new idea, instead are added.

Same happens with the economic tree, focusing on the "Expand Oil Industry", the final option "Achieving Venezuelan Autarky" doesnt replace "Venezuelan Oil" but instead adds the modifiers.

**Comment by TheBrit7** [comment]
Thanks for reporting, I'll go take a look

**Comment by TheBrit7** [comment]
Could you please describe and pinpoint (preferably in detail) what you are doing or how you think this may of happened? I have tested this about four times 4 now and it has not happened at all.

**Comment by hurafloyd** [comment]
Will start a new game now. I'll get back to you in a bit. I had a game yesterday where this happened, will try to reproduce it as much as possible.

**Comment by hurafloyd** [comment]
I definitely misinterpreted the description, I got confused by the "replace" in it, I thought it meant that they were supposed to be replaced by the new spirit. Sorry for the hassle.

It might be a good idea to change the description, I just checked the chinese tree and the focuses there do explain a modification to an existing condition by a certain margin. For example, the "National Reform" focus modifies the "Corrupted Economy" law as explained in the description.
